scripts/jenkins/ardana/gerrit/gerrit_merge.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
+from gerrit import GerritChange  # noqa: E402
+from gerrit_project_map import gerrit_project_map  # noqa: E402
+
+
+def check_dependencies(change):
```

Comment:
(nit) perhaps call this method "check_all_dependencies_satisfied"

Reply:
done